### ISSUE TITLE
Docker image 

### ISSUE THREAD

**Issue by SolidHak** [nelsonmaligro/e-Dokyumento#9]
Hello, is docker image is going to be support One Day ?

Thanks for the feedback.

**Comment by nelsonmaligro** [comment]
Yes @SolidHak  we will work out on the docker image. But for now you can try the Installation procedure on Linux; it's much easier.
https://github.com/nelsonmaligro/e-Dokyumento/blob/master/docs/Install%20e-Dokyumento%20on%20Ubuntu%20Linux.pdf

**Comment by nelsonmaligro** [comment]
> Hello, is docker image is going to be support One Day ?
> 
> Thanks for the feedback.

Docker is now supported
Simple invoke: docker-compose up --build

**Comment by SolidHak** [comment]
Many thanks for your effort
